### PR TITLE
Update harbour-seaprint-fr.ts

### PR DESCRIPTION
**Pull request by phklrz**

### CODE REVIEW

File: translations/harbour-seaprint-fr.ts [hunk 1]
```diff
@@ -177,31 +177,31 @@
...
     </message>
     <message>
         <source>Failed to open file</source>
-        <translation type="unfinished"></translation>
+        <translation>Échec de l&apos;ouverture du fichier</translation>
     </message>
     <message>
         <source>Printing</source>
-        <translation type="unfinished"></translation>
+        <translation>Impression en cours d&apos;</translation>
```
Comment (attah) [inline translations/harbour-seaprint-fr.ts]: I believe this goes on the "busy page", and i'm getting the impression you tried out the app with this translation.
But i still feel like i should ask - so you want this line to just end in " d' "? 
(and then the percentage shows up bigger on the next line)

File: translations/harbour-seaprint-fr.ts [hunk 2]
```diff
@@ -309,38 +309,38 @@
...
     <name>IppPrinter</name>
     <message>
         <source>Failed to open file</source>
-        <translation type="unfinished"></translation>
+        <translation>Échec de l&apos;ouverture du fichier</translation>
     </message>
     <message>
         <source>Unsupported print media</source>
-        <translation type="unfinished"></translation>
+        <translation>média d&apos;impression non supporté</translation>
```
Comment (attah) [inline translations/harbour-seaprint-fr.ts]: Capitalization?

File: translations/harbour-seaprint-fr.ts [hunk 3]
```diff
@@ -441,142 +441,142 @@
...
     </message>
     <message>
         <source>Media handling</source>
-        <translation type="unfinished"></translation>
+        <translation>Gestion du média</translation>
     </message>
     <message>
         <source>Reset default settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Réinitialiser aux paramètres par défaut</translation>
```
Comment (attah) [inline translations/harbour-seaprint-fr.ts]: This is at risk of being too long, should i check or did you do that?

File: translations/harbour-seaprint-fr.ts [hunk 4]
```diff
@@ -659,107 +659,107 @@
...
     </message>
     <message>
         <source>auto-sense</source>
-        <translation type="unfinished"></translation>
+        <translation>détection automatique</translation>
     </message>
     <message>
         <source>by-pass tray</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (attah) [inline translations/harbour-seaprint-fr.ts]: empty... bac auxiliaire?

**Review by attah — COMMENTED** [review]
Thank you!
From what little i can tell, this looks like a very good quality translation.

I have added a few comments, and there are a few untranslated strings.
Let me know if you want to fix any of it - or if i should go ahead and just merge.
Plenty good enough to do that, but even a translation-sceptic like myself is starting to look at just googling the last few things for completeness. (and we all know how that goes)

Edit: and let me know if i should (try to) explain any of the terms.

File: translations/harbour-seaprint-fr.ts (hunk 1 above)
Comment (phklrz) [reply translations/harbour-seaprint-fr.ts]: I rephrase my translation and forgot to remove the " d\&apos; ".

File: translations/harbour-seaprint-fr.ts (hunk 2 above)
Comment (phklrz) [reply translations/harbour-seaprint-fr.ts]: Yes.

File: translations/harbour-seaprint-fr.ts (hunk 4 above)
Comment (phklrz) [reply translations/harbour-seaprint-fr.ts]: Ok.

File: translations/harbour-seaprint-fr.ts (hunk 3 above)
Comment (phklrz) [reply translations/harbour-seaprint-fr.ts]: I did it. Reviewing it, it should be "Réinitialiser les paramètres par défaut". I don't see how I could shorten it in French.

**Comment by attah** [comment]
Are you planning another revision, or do you want me to address the comments after merge?

**Comment by phklrz** [comment]
I'm planning a new revision

**Comment by attah** [comment]
Looks good, merging.
Thanks for your contribution.

**Pull request merged**